src/api.py

```diff
@@ -48,7 +49,13 @@ def get_events(event_name, token=None, limit=None):
     issue_id = ISSUES[event_name]
 
     # Initiate session
-    db_client = MongoClient()
+    db_client = (
+        MongoClient() if mongo_password is None
+        else MongoClient(
+            f"mongodb+srv://{quote_plus(mongo_user)}:{quote_plus(mongo_password)}"
+            "@sentrybackup.sywajav.mongodb.net/?retryWrites=true&w=majority"
+        )
+    )
```

Comment:
Rather than calling `MongoClient` twice, what about having host default to `"localhost"` and then updating it if user credentials are present?